libs/feature/search/src/lib/filter-dropdown/filter-dropdown.component.spec.ts

```diff
@@ -47,6 +49,14 @@ class FieldsServiceMock {
   getFieldType = jest.fn(() => 'values')
 }
 
+const configFilters: SearchFilters = {
```

Comment:
Inconsistency in the type, you used `any` above, and it should be `FieldFilters`